taskcluster/test-training-extra_16k-linux-amd64-py35m-opt.yml

```diff
@@ -1,13 +0,0 @@
-build:
```

Comment:
no more py35 support I gues ?

Reply:
Not supported by tensorflow_addons and pip itself will drop support soon. We already documented 3.6 as a req so I just deleted the tests.

Reply:
Thanks, I have not had a look to the 2.3 reqs :)